scalyr_agent/util.py

```diff
@@ -159,9 +159,9 @@ def json_dumps_custom(obj, fp):
 
             if fp is not None:
                 # Eliminate spaces by default. Python 2.4 does not support partials.
-                return json.dump(obj, fp, sort_keys=True, separators=(",", ":"))
+                return json.dump(obj, fp)
```

Comment:
I don't quite remember the spaces by default comment here.. But, I will say, any place where we were doing something to support Python 2.4 or 2.5, we can kill now.